workflow-bot-app/src/main/java/com/symphony/bdk/workflow/api/v1/dto/StatusEnum.java

```diff
@@ -2,9 +2,22 @@
 
 public enum StatusEnum {
   PENDING,
-  COMPLETED;
+  COMPLETED,
+  FAILED;
 
   public static StatusEnum toInstanceStatusEnum(String status) {
-    return "ACTIVE".equals(status) ? StatusEnum.PENDING : StatusEnum.COMPLETED;
+    if (status == null) {
```

Comment:
i think we can first make the input value all capital, and use StatusEnum.valueOf() to get the instance. it will be much simpler, and the exception is handled also inside the valueOf method

Reply:
The issue here is that we need to map both "ACTIVE" and "PENDING" to StatusEnum.PENDING. In Camunda's point of view, a process can be "active" while in WDK POV it is "PENDING".  
I also prefer to mention the valid values in the error which is not provided in the exception returned by valueOf(). WDYT?